lib/Controller/SAMLController.php

```diff
@@ -418,8 +418,14 @@ public function singleLogoutService() {
 			$stay = true ; // $auth will return the redirect URL but won't perform the redirect himself
 			if ($isFromIDP) {
 				$keepLocalSession = true ; // do not let processSLO to delete the entire session. Let userSession->logout do the job
```

Comment:
Is it art or may we remove it?

Reply:
Wait. I did a mistake here.... :/

Reply:
I should have replaced the third parameter but replaced the first one 🙈 - :/ :/ :/ - That's what you get for copy-pasting from one system to another.

Reply:
Background: I am unable to test this locally since the IDPs supporting this all cost $$$. And when I copy-pasted it from the support ticket, I seem to have put it into the wrong line. Sorry, and good catch!